flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/utils/Expander.java

```diff
@@ -70,30 +75,43 @@ public static Expander create(FlinkPlannerImpl planner) {
 	/** Expands identifiers in a given SQL string, returning a {@link Expanded}. */
 	public Expanded expanded(String ori) {
 		final Map<SqlParserPos, SqlIdentifier> identifiers = new HashMap<>();
+		final Map<String, SqlIdentifier> funcNameToId = new HashMap<>();
```

Comment:
Why did you decide to use a different key for functions? Can't we use the `SqlParserPos` as well?

Reply:
Because the `BridgingSqlFunction` lost the parser position information.